compat.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * libtinyiiod - Tiny IIO Daemon Library
+ *
+ * Copyright (C) 2016 Analog Devices, Inc.
+ * Author: Paul Cercueil <paul.cercueil@analog.com>
```

Comment:
you can put yourself as author here and update the year to 2019
and change `libtinyiiod - Tiny IIO Daemon Library`  ->  `libc compatibility layer` or something like that